app/views/comments/_comment.html.erb

```diff
@@ -124,7 +124,7 @@ class="comment <%= comment.current_vote ? (comment.current_vote[:vote] == 1 ?
           <% if comment.flags > 0 &&
           comment.show_score_to_user?(@user) &&
           (comment.user_id == @user.try(:id) || @user.try("is_moderator?")) %>
-            | <%= comment.vote_summary_for_user(@user).downcase %>
+            <%= comment.vote_summary_for_user(@user).downcase %>
```

Comment:
In one sense, this patch simply draws attention to the display bug by changing the lines in question. As far as I understand it, the problem was introduced in commit e1537b0 with that work following a score related regression fixed in commit f377fd.

In this case, I'd appreciate a more idiomatic or straightforward fix.